plugins/modules/ndo_virtual_port_channel_interface.py

```diff
@@ -0,0 +1,383 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Gaspard Micol (@gmicol) <gmicol@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = r"""
+---
+module: ndo_virtual_port_channel_interface
+short_description: Manage Virtual Port Channel Interfaces on Cisco Nexus Dashboard Orchestrator (NDO).
+description:
+- Manage Virtual Port Channel Interfaces on Cisco Nexus Dashboard Orchestrator (NDO).
+- This module is only supported on ND v3.2 (NDO v4.4) and later.
+author:
+- Gaspard Micol (@gmicol)
+options:
+    template:
+        description:
+        - The name of the template.
+        - The template must be a fabric resource template.
+        type: str
+        required: true
+    virtual_port_channel_interface:
+        description:
+        - The name of the Virtual Port Channel Interface.
+        type: str
+        aliases: [ name, virtual_port_channel, vpc ]
+    virtual_port_channel_interface_uuid:
+        description:
+        - The UUID of the Virtual Port Channel Interface.
+        - This parameter is required when parameter O(virtual_port_channel_interface) is updated.
+        type: str
+        aliases: [ uuid, virtual_port_channel_uuid, vpc_uuid ]
+    description:
+        description:
+        - The description of the Virtual Port Channel Interface.
+        type: str
+    node_1:
+        description:
+        - The first node ID.
+        type: str
+    node_2:
+        description:
+        - The second node ID.
+        type: str
+    interfaces_node_1:
+        description:
+        - The list of used Interface IDs for the first node.
+        - Ranges of Interface IDs can be used.
+        type: list
+        elements: str
+        aliases: [ interfaces_1, members_1 ]
+    interfaces_node_2:
+        description:
+        - The list of used Interface IDs for the second node.
+        - Ranges of Interface IDs can be used.
+        type: list
+        elements: str
+        aliases: [ interfaces_2, members_2 ]
+    interface_policy_group_uuid:
+        description:
+        - The UUID of the Port Channel Interface Setting Policy.
+        type: str
+        aliases: [ policy_uuid, interface_policy_uuid ]
+    interface_policy_group:
+        description:
+        - The name of the Port Channel Interface Policy Group.
+        - This parameter can be used instead of O(interface_policy_group_uuid).
+        type: dict
+        suboptions:
+            name:
+                description:
+                - The name of the Port Channel Interface Setting Policy.
+                type: str
+            template:
+                description:
+                - The name of the template in which is referred the Port Channel Interface Policy Group.
```

Comment:
this sentence is a bit odd for me to read, could you rephrase?

Reply:
sure, will rephrase it.